shell/client/setup-wizard/wizard.html

```diff
@@ -44,7 +44,7 @@
                                    mayJumpTo=(mayJumpTo "identity")
                                    route="setupWizardIdentity"
                                    className=itemClassName}}
-      Login provider(s)
+      Identity providers
```

Comment:
One other thing:

> I don't think we've received criticism of the term "identity" being problematic; only that inconsistency in copy is confusing.

I think @neynah above says that we have received criticism of the term "identity" being confusing, unless I'm misunderstanding her.

Regardless I'm still +1 on making this discussion not be a blocker for this particular PR. Let's take the time to make a careful decision, and until then, carefully make no decision. The places where this PR changes the string are OK with me; they increase consistency.

I filed https://github.com/sandstorm-io/sandstorm/issues/2584 to track that conversation.